client/js/compute.js

```diff
@@ -684,6 +684,80 @@ const compute_ops = [
     sample: 'var a = randRange ${x} ${y} ${z}',
     call: function(v, x, y, z) { return v = Math.round(Math.floor((Math.random() * (y - x) / (z || 1))) * (z || 1) + x) },
     hash: '4758cba8f404cfe3ef0a300cebf3938a'
+  },
+  {
+    name: 'colorMono',
+    desc: 'returns a color value of #000000 or #ffffff, whichever provides better contrast to the provided color',
+    sample: 'var a = colorMono ${x}',
+    call: function(v, x) {
+      // Code created by https://chat.openai.com/chat
+      let r = parseInt(x.slice(1, 3), 16);
+      let g = parseInt(x.slice(3, 5), 16);
+      let b = parseInt(x.slice(5, 7), 16);
+      let luminance = (0.2126 * r + 0.7152 * g + 0.0722 * b) / 255;
+      return v = (luminance <= 0.5) ? '#ffffff' : '#000000';
+    },
+    hash: '123'
+  },
+  {
+    name: 'colorHue',
+    desc: 'returns a hex color value in the same hue but different luminance of the provided color to provide contrast',
```

Comment:
Do you want to describe the parameters here, or is that left to the wiki?

Reply:
I think I should add the input and output to be consistent with the other 2.  It is also in the draft wiki entry I just submitted.